taskcluster/ci/docker-image/kind.yml

```diff
@@ -13,3 +13,6 @@ transforms:
 jobs:
     base:
         symbol: I(base)
+    ui-tests:
+        parent: base
+        symbol: I(ui-tests)
```

Comment:
does this give us treeherder reporting?  I'm adding this to fenix, but wanted to eventually make sure UI test results post to treeherder as well (not sure how this is done in taskgraph)

Reply:
This adds treeherder reporting for the docker image build. The `treeherder:` stanza below does for the tests.

Reply:
To be more explicit, the treeherder reporting on the test jobs themselves is done here: https://github.com/mozilla-mobile/reference-browser/pull/892/files#diff-24c9b9fc54c62f21ea7365d5b268f925R49